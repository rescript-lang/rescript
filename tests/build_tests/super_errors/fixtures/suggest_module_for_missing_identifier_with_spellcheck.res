let consol = 1
console.log("Hello")
