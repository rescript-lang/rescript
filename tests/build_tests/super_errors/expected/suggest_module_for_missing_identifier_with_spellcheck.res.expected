
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/suggest_module_for_missing_identifier_with_spellcheck.res[0m:[2m2:1-7[0m

  1 [2mâ”‚[0m let consol = 1
  [1;31m2[0m [2mâ”‚[0m [1;31mconsole[0m.log("Hello")
  3 [2mâ”‚[0m 

  The value console can't be found
  
  [1;33mHint: Did you mean consol?[0m

  Or did you mean to use the module [1;33mConsole[0m?